<ui version="4.0" >
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Настройки recognitionPlugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="1" column="2" >
    <widget class="QLineEdit" name="lineEdit_pauseModel" />
   </item>
   <item row="1" column="4" >
    <widget class="QToolButton" name="toolButton_setPauseModel" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="4" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="13" column="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Модель паузы:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Директория файлов для обработки:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" >
    <widget class="QLineEdit" name="lineEdit_processingDir" />
   </item>
   <item row="7" column="4" >
    <widget class="QToolButton" name="toolButton_setProcessingDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" >
    <widget class="QLabel" name="label_8" >
     <property name="text" >
      <string>Директория файлов отчётов:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" >
    <widget class="QLineEdit" name="lineEdit_reportsDir" />
   </item>
   <item row="8" column="4" >
    <widget class="QToolButton" name="toolButton_setReportsDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" >
    <widget class="QLabel" name="label_15" >
     <property name="text" >
      <string>Максимальное количество потоков:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QSpinBox" name="spinBox_maxBPThreadCount" >
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="1" colspan="4" >
    <widget class="QCheckBox" name="checkBox_useExternalProcessing" >
     <property name="text" >
      <string>Использовать внешний поток для рассчёта</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="4" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Модель речи:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLineEdit" name="lineEdit_speechModel" />
   </item>
   <item row="2" column="4" >
    <widget class="QToolButton" name="toolButton_setSpeechModel" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLineEdit" name="lineEdit_noiceModels" />
   </item>
   <item row="3" column="4" >
    <widget class="QToolButton" name="toolButton_setNoiceModes" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Модели шума:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Модели языков:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QLineEdit" name="lineEdit_languagesModels" />
   </item>
   <item row="4" column="4" >
    <widget class="QToolButton" name="toolButton_setLanguagesModels" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Модели фонем:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QLineEdit" name="lineEdit_phonemModels" />
   </item>
   <item row="5" column="4" >
    <widget class="QToolButton" name="toolButton_setPhonemModels" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
